mvpa2/algorithms/group_clusterthr.py

```diff
@@ -260,13 +264,13 @@ def featuresegment_producer(ncols):
                 # one average map for every stored bcombo
                 # this also slices the input data into feature subsets
                 # for the compute blocks
-                yield [np.mean(
+                yield np.array([np.mean(
```

Comment:
On Tue, 23 Feb 2016, Nikolaas N. Oosterhof wrote:

>    In mvpa2/algorithms/group_clusterthr.py:

>  > @@ -260,13 +264,13 @@ def featuresegment_producer(ncols):
>  >                  # one average map for every stored bcombo
>  >                  # this also slices the input data into feature subsets
>  >                  # for the compute blocks
>  > -                yield [np.mean(
>  > +                yield np.array([np.mean(

>    Why the np.array constructor?

it seems that sending a single array through the pipe to joblib is
faster than sending a list of arrays which later get arrayed (in our
experimentation)